assignment_3.rb

```diff
@@ -0,0 +1,36 @@
+def solution(a,b,c) 
+  s = ''
+  count = 0
+  total = a + b + c 
+  hash = { 'a'=> a, 'b'=> b, 'c'=> c}
+  hash_sort = hash.sort_by { |k, v| -v}
+  k_max, v_max = hash_sort.first 
+  k_mid, v_mid = hash_sort[1]
+  k_min, v_min = hash_sort.last
```

Comment:
Function `solution` is a bit too long. Can we extract this to a new function and call it from here?